src/popup/components/AddressBook/AddressBookList.vue

```diff
@@ -131,9 +171,27 @@ export default defineComponent({
       emit('update:hideButtons', hideOuterButtons.value);
     }, 100);
 
+    function accountIcon(type: AccountType) {
+      switch (type) {
+        case ACCOUNT_TYPES.airGap:
+          return AirGapIcon;
+        case ACCOUNT_TYPES.privateKey:
+          return PrivateKeyIcon;
+        default:
+          return null;
+      }
+    }
+    function bgColorStyle(isAddressBookEntry: boolean, isChainName: boolean, address: String) {
```

Comment:
I suggest to add empty lines between function definitions to follow common pattern within our app :)